xivo/tests/test_auth_verifier.py

```diff
@@ -108,9 +108,7 @@ def decorated():
 
         decorated()
 
-        mock_client.token.get.assert_called_once_with(s.token)
```

Comment:
Why is this assertion removed?

Reply:
it made sense in my my head because the token content could be cached. but given the context of this test the call will still happen. I'm putting it back

Reply:
It made sense in my head because the token content could be cached. But in this context it's always going to be called. Adding it back